broadlink/__init__.py

```diff
@@ -63,7 +63,9 @@ def gendevice(devtype, host, mac, name=None, cloud=None):
         S1C: [0x2722],  # S1 (SmartOne Alarm Kit)
         dooya: [0x4E4D],  # Dooya DT360E (DOOYA_CURTAIN_V2)
         bg1: [0x51E3], # BG Electrical Smart Power Socket
-        lb1 : [0x60c8]   # RGB Smart Bulb
+        lb1 : [0x60c8], # RGB Smart Bulb
+        bsl1 : [0x5043] # OEM branded BSL1
```

Comment:
The same class could be used, but I have not managed to change the BSL's brightness or color temperature with the LB1 class, since the BSL1 lacks the RGB functionallity of the LB1. The BSL1 class is a copy of the LB1 class with one added function, if the function could be implemented in the LB1 class one could utilize the full functionallity of the BSL1 with LB1 class.